components/global.test.js

```diff
@@ -0,0 +1,97 @@
+import { shallow } from "enzyme";
+import   {MyApp } from "../pages/_app"
+import AppHead from "../Utils/layouts/head";
+import { PageFooter} from "../components/footer"
+import BreadCrumb, {BreadAnchor, BreadLink} from "../Utils/layouts"
+import * as nextRouter from 'next/router';
+
+
+describe("AppJs", () => {
+    it("renders footer component", () =>{
+        const app = shallow(<MyApp />).childAt(0).dive();
+        expect(app.containsMatchingElement(
+                <PageFooter/>
+        )).toBe(true);    
+    })
+    it("renders header component", () =>{
```

Comment:
`renders header component` appears to be testing the document `<head>` rather than the header component. Could we make this clearer in the description?

Reply:
👍